charts/osm/crds/multiclusterservice.yaml

```diff
@@ -40,9 +40,13 @@ spec:
               required:
                 - serviceAccount
                 - clusters
+                -protocol
               properties:
                 serviceAccount:
                   type: string
+                protocol:
+                  description: The IP protocol for this port. Supports "TCP", "UDP", and "SCTP". Default is TCP.
```

Comment:
If you want a fixed set of allowed values please follow this example : https://github.com/openservicemesh/osm/blob/f9dc7b9be2287303345154d5b490763d3a051e3e/charts/osm/crds/meshconfig.yaml#L54

Further please define a default as well in the CRD

Reply:
This hasn't yet been addressed for the protocol

Reply:
done!